### ISSUE TITLE
No Active Ratings Error

### ISSUE THREAD

**Issue by JasonDSS** [opendcs/opendcs#317]
**Describe the bug**
I keep getting a "No Active Ratings" error when I try to run a computation in CCP that uses a rating defined in CWMS DB. The rating spec does exist in CWMS and it is checked as active. Please see screenshots below:

**To Reproduce**
1. Create a Python CCP algorithm and computation that uses a rating defined in CWMS.
2. No storage output after running comp
3. Trace Execution shows that there's no active rating
4. Please see screenshots below

**Expected behavior**
There should be storage output

**Screenshots**
![image](https://github.com/opendcs/opendcs/assets/143829980/dbbf4727-183b-4c97-8c63-b7c61014a56b)
![image](https://github.com/opendcs/opendcs/assets/143829980/747913b6-3cd9-497c-8f41-badfac52bf9c)
![image](https://github.com/opendcs/opendcs/assets/143829980/bbe944e3-006e-41c5-bc74-54bc632d924f)
![image](https://github.com/opendcs/opendcs/assets/143829980/18be770a-be0e-481e-a1d2-25d0f124e3f1)

**Desktop/ServerSide (please complete the following information):**
 - OS: Windows 10
 - Version: OpenDCS 7.0.8 GUI Fix (client side)

**Additional context**

**Comment by JasonDSS** [comment]
So I was finally able to get it to run, but I had to create another rating in CWMS DB. Not sure why the previous rating didn't work, but I suspect it has to do with the "ERROR" values in the dependent variable column under the "Expanded" tab in the CWMS Rating Editor. No matter what I did, I just could not get the "ERROR" values to populate, so I just started over and created a new rating, which seemed to populate just fine. So this seems to be a CWMS-VUE issue rather than an OpenDCS issue.

Suspected cause of the "No Active Ratings" issue: ERROR values in the dependent variable column:
![image](https://github.com/opendcs/opendcs/assets/143829980/fe192f21-0f32-4c76-b585-6c54239b634c)
